relayer/src/link/relay_path.rs

```diff
@@ -527,7 +527,7 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> RelayPath<ChainA, ChainB> {
     ///
     /// Side effects: may schedule a new operational data targeting the source chain, comprising
     /// new timeout messages.
-    fn regenerate_operational_data(
+    pub(crate) fn regenerate_operational_data(
```

Comment:
The doc for this method assumes that it should only be called from `relay_from_operational_data` (L520). Can we update the doc to reflect the fact that `regenerate_operational_data` has now multiple uses?